packages/apps/public/locales/zh/app-settings.json

```diff
@@ -16,6 +16,7 @@
   "default interface language": "默认接口语言",
   "Cancel": "取消",
   "Save & Reload": "保存并重载",
+  "Export Network Specs": "导出网络配置",
```

Comment:
Thanks an absolute million for this! The cn translation is lagging atm, after the initial push, there has been no updates. :(

Reply:
I will open another PR to add some Chinese Translations, and keep this PR focus.